sample_configs/values-pyroscope-capz.yaml

```diff
@@ -0,0 +1,825 @@
+# This config will allow the agent to scrape both ebpf and user-space profiles.
+
+psp:
+  enabled: true
+kyvernoPolicyExceptions:
+  enabled: true
+  namespace: giantswarm
+grafana-agent:
+  controller: 
+    hostPID: true
+  crds:
+    create: false
+  podAnnotations:
+    profiles.grafana.com/memory.scrape: "true"
+    profiles.grafana.com/memory.port_name: "http-metrics"
+    profiles.grafana.com/cpu.scrape: "true"
+    profiles.grafana.com/cpu.port_name: "http-metrics"
+    profiles.grafana.com/goroutine.scrape: "true"
+    profiles.grafana.com/goroutine.port_name: "http-metrics"
+  agent:
+    securityContext:
+      privileged: true
+      runAsGroup: 0
+      runAsUser: 0
+    configMap:
+      create: true
+      content: |
+        logging {
+          level  = "info"
+          format = "logfmt"
+        }
+
+        discovery.kubernetes "pyroscope_kubernetes" {
+          selectors {
+            field = "spec.nodeName=" + env("HOSTNAME")
+            role = "pod"
+          }
+          role = "pod"
+        }
+        
+        pyroscope.ebpf "instance" {
+          forward_to = [pyroscope.write.pyroscope_write.receiver]
+          targets = discovery.kubernetes.pyroscope_kubernetes.targets
+        }
+        
+        // The default scrape config allows to define annotations based scraping.
+        //
+        // For example the following annotations:
+        //
+        // ```
+        // profiles.grafana.com/memory.scrape: "true"
+        // profiles.grafana.com/memory.port: "8080"
+        // profiles.grafana.com/cpu.scrape: "true"
+        // profiles.grafana.com/cpu.port: "8080"
+        // profiles.grafana.com/goroutine.scrape: "true"
+        // profiles.grafana.com/goroutine.port: "8080"
+        // ```
+        //
+        // will scrape the `memory`, `cpu` and `goroutine` profiles from the `8080` port of the pod.
+        //
+        // For more information see https://grafana.com/docs/phlare/latest/operators-guide/deploy-kubernetes/#optional-scrape-your-own-workloads-profiles
```

Comment:
Replace link with this
https://grafana.com/docs/pyroscope/latest/deploy-kubernetes/helm/#optional-scrape-your-own-workloads-profiles

Reply:
Done upstream :)